src/HelperPluginManager.php

```diff
@@ -341,6 +58,9 @@ class HelperPluginManager extends AbstractPluginManager
      */
     public function __construct($configOrContainerInstance = null, array $v3config = [])
     {
+        $this->aliases   = ConfigProvider::defaultViewHelperAliases();
```

Comment:
I haven't checked this due to laziness, but the aliases which were registered must not change in any way. Juggling between aliases/factories introduces BC breaks as we learned here:

https://github.com/laminas/laminas-mvc/issues/99

If you can confirm that the `factories` and the `aliases` are 100% the same as in the previous version, we're fine here.

Reply:
See: https://github.com/laminas/laminas-view/pull/110#discussion_r803644583